projectroles/tests/test_commands.py

```diff
@@ -495,6 +501,18 @@ def _make_undefined_setting(self):
         ghost.save()
         return ghost
 
+    def _make_setting_category_type(self):
+        """Create database setting with category type project"""
+        catsett = AppSetting(
```

Comment:
Minor point, but note the variable naming. Words should be separated by underscores as per convention :)